docs/admin-guide.md

```diff
@@ -17,29 +17,51 @@ To get started, a plugin that is installed on the Mattermost Server is needed to
+6. Choose **Settings** at the bottom of the page, enable development mode, and apply the changes. Development mode allows you to install apps from outside of the Atlassian Marketplace.
+7. Press **Upload app**.
 
     ![image](https://github.com/mattermost/mattermost-plugin-confluence/assets/74422101/158bd7b7-4e36-41d5-872a-def2f617213f)
 
-6. Upload the OBR file you downloaded earlier. 
-7. Once the app is installed, press **Configure** to open the configuration page.
-8. In the **Webhook URL field**, enter the URL that is displayed after you typed `/confluence install server` - it is unique to your server. You'll need to copy/paste it from Mattermost into the **From this URL** field in Confluence. 
-9. Press **Save** to finish the setup.
-10. Go to **Settings > Apps > Manage Apps**.
-11. Choose **Settings** at the bottom of the page, enable development mode, and apply the change. Development mode allows you to install apps from outside of the Atlassian Marketplace.
+8. Upload the OBR file you downloaded earlier. 
+9. Once the app is installed, press **Configure** to open the configuration page.
+10. In the **Webhook URL field**, enter the URL that is displayed after you typed `/confluence install server` - it is unique to your server. You'll need to copy/paste it from Mattermost into the **From this URL** field in Confluence. 
+11. Press **Save** to finish the setup.
+12. Go to **Settings > Apps > Manage Apps**.
+13. Choose **Settings** at the bottom of the page, enable development mode, and apply the change. Development mode allows you to install apps from outside of the Atlassian Marketplace.
 
     ![image](https://github.com/mattermost/mattermost-plugin-confluence/assets/74422101/fa27854e-6305-4164-963f-a5692284bf95)
 
-12. Once installed, you will see the "Installed and ready to go!" message in Confluence.
-13. You can now go to Mattermost and type `/confluence subscribe` in the channel you want to get notified by Confluence.
+14. Once installed, you will see the "Installed and ready to go!" message in Confluence.
+15. You can now go to Mattermost and type `/confluence subscribe` in the channel you want to get notified by Confluence.
+
+#### Set up Confluence Server for server version greater than or equal to 9
+
+To get started, type in `/confluence install server` in a Mattermost chat window.
+
+1. Provide the Confluence base URL in the first step of the installation
+2. Select **No** in the **Confluence server version greater than or equal to 9** question in installation.
+3. Log in as an Administrator on Confluence to configure the plugin.
+3. Create a new application link in your Confluence server by going to **Settings > Applications > Application Links**
+
+    ![image](https://user-images.githubusercontent.com/90389917/202149868-a3044351-37bc-43c0-9671-aba169706917.png)
+
+4. Select **Create link**
```

Comment:
Please be consistent with periods on bullet points. I'm 0/5 on whether bullet points get periods, but it must be consistent.

Reply:
Most of the bullet points in the list are in continuation, so let's remove all the periods

Reply:
Most of the bullet points in the lists are in continuation so let's remove the periods from all of the bullet points

Reply:
Numbered bullet points should use either a point or a parenthesis: https://github.github.com/gfm/#lists